src/SchemaDesign/QuestionPicker.tsx

```diff
@@ -0,0 +1,84 @@
+import Dialog from '@mui/material/Dialog'
+import DialogTitle from '@mui/material/DialogTitle'
+import React from 'react'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import Accordion from '@mui/material/Accordion'
+import Typography from '@mui/material/Typography'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import DialogContent from '@mui/material/DialogContent'
+import TextFieldsIcon from '@mui/icons-material/TextFields'
+import Stack from '@mui/material/Stack'
+import CheckBoxIcon from '@mui/icons-material/CheckBox'
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth'
+import Icon from '@mui/material/Icon'
+import { SchemaQuestion } from '@/types/interfaces'
+import DialogContentText from '@mui/material/DialogContentText'
+import SchemaDesignerQuestion from './SchemaDesignerQuestion'
+
+export default function QuestionPicker({
+  onSubmit,
+  handleClose,
+  questionPickerOpen,
+}: {
+  onSubmit: (data: SchemaQuestion) => void
+  handleClose: () => void
+  questionPickerOpen: boolean
+}) {
+  const [expanded, setExpanded] = React.useState('')
+
+  const handleAccordianChange = (panel: string) => {
+    setExpanded(panel)
+  }
+
+  const questionList = [
```

Comment:
This can be moved outside of the component. It doesn't need to be recreated on every render.